exchanges/bybit/bybit_wrapper.go

```diff
@@ -606,7 +601,7 @@ func (by *Bybit) UpdateTicker(ctx context.Context, p currency.Pair, assetType as
 			return nil, err
 		}
 
-		cp, err := by.extractCurrencyPair(tick.Symbol, assetType)
+		cp, err := by.MatchSymbolWithAvailablePairs(tick.Symbol, assetType, false)
```

Comment:
Same with all of these. You need to be filtering with enabled. I'm fine with how you did it with Bitmex just using a `continue`, but if you end up making a `MatchSymbolWithEnabledPairs`, that's cool too